src/NSubstitute.Analyzers.Shared/DiagnosticDescriptors.cs

```diff
@@ -12,6 +12,8 @@ internal class DiagnosticDescriptors<T>
                 $"{typeof(T).GetTypeInfo().Assembly.GetName().Name}.Resources",
                 typeof(T).GetTypeInfo().Assembly);
 
+        private static string helpLinkUriFormat = "https://github.com/nsubstitute/NSubstitute.Analyzers/blob/master/documentation/{0}.md";
```

Comment:
Should this be `blob/master/documentation/rules/{0}.md` instead? (add `/rules/` part of path?)

Reply:
yes, fixed